feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/task.c

```diff
@@ -183,37 +200,6 @@ static int command_conf_add(struct schema_Command_Config *conf)
 	return -1;
 }
 
-static int command_conf_del(struct schema_Command_Config *conf)
```

Comment:
I think the old version was an empty function.  Can you just do that?

Reply:
done